web_search_with_and/static/src/js/search_bar.esm.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
```

Comment:
I think this file can be removed: https://github.com/odoo/odoo/pull/142858

Reply:
Updated, Thanks